consts.go

```diff
@@ -9,8 +9,21 @@ const (
 )
 
 const (
-	pageStatusOther    = 0
-	pageStatusWritten  = 1
-	pageStatusComitted = 2
-	pageStatusApplied  = 3
+	pageStatusInvalid = iota
+	pageStatusOther
+	pageStatusWritten
+	pageStatusComitted
+	pageStatusApplied
+)
+
+// Metadata contains the header and version of the data being stored.
+type Metadata struct {
+	Header, Version string
+}
+
+var (
+	metadata = Metadata{
+		Header:  "WAL",
+		Version: "1.0",
+	}
```

Comment:
idiomatic ordering within Nebulous repos is consts, then vars, then types. It shouldn't matter to golang that the type is defined after a var declares usage of it, I believe the compiler handles that gracefully.

The metadata needs some docstring. Even if it's just `// metadata defines the WAL v1.0.0 metadata`.